tools/contributors-requirements.txt

```diff
@@ -0,0 +1,6 @@
+# These are pip requirements for contributors.py. They were extracted from
```

Comment:
Let's remove this file. Looks it it got cherry-picked from the other PR, where review suggested that it not be included.